<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGraphicsDialog</class>
 <widget class="QDialog" name="ViewGraphicsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit plots</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxProfile">
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxLineScintillators">
     <property name="text">
      <string>Line Scintillators</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxValueScintillators">
     <property name="text">
      <string>Value Scintillators</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMonitoringGalo">
     <property name="text">
      <string>Monitoring Galo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
